RFCs/0026-DataCollector Extensibility - Getting testSources.md

```diff
@@ -0,0 +1,70 @@
+# 0026 DataCollector Extensibility - Getting testSources in property bag 
+
+## Summary
+Getting `testSources` as part of property bag. List of test sources can be used by datacollectors for processing before test run start.
+
+## Motivation
+Data collector might need test sources list while initialization. Example: Static code coverage data collector needs to instrument the test sources before test run start.
+
+In `sessionStartEventArgs` the field `IDictionary<string, object> Properties` will contain all properties we need to pass to the datacollector.
```

Comment:
SessionStartEventArgs will contains a test platform properties bag. Currently TestPlatform passes following properties to the datacollector extensions:
1. "TestSources" : IEnumerable<String>
    TestSources is enumberable of testSources passed by used in test run.